sdk/identity/azure-identity/README.md

```diff
@@ -76,7 +76,7 @@ Use the [Azure CLI][azure_cli] snippet below to create/get client secret credent
     }
     ```
 
-* Use the returned credentials above to set  **AZURE_CLIENT_ID**(appId), **AZURE_CLIENT_SECRET**(password) and **AZURE_TENANT_ID**(tenant) [environment variables](#environment-variables).
+* Use the returned credentials above to set  **AZURE\_CLIENT\_ID**(appId), **AZURE\_CLIENT\_SECRET**(password) and **AZURE\_TENANT\_ID**(tenant) [environment variables](#environment-variables).
```

Comment:
This change shouldn't be in here, you'd fixed this as part of [another PR](https://github.com/Azure/azure-sdk-for-java/pull/11207). You should probably rebase with master.